docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -119,6 +119,42 @@ This property is optional; the default is ``1m``.
 Enable or disable using the address published by Elasticsearch to connect for
 queries.
 
+``elasticsearch.union-schema-indices-for-alias``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enable or disable merging of schemas of indices for governing the schema of an alias.
+If disabled, schema of the alias follows the schema of first index only.
+Same field with different data types in different indices for an alias casts
+as :ref:`raw JSON <elasticsearch-raw-json-transform>`.
+
+This property is optional; the default value is ``false``.
```

Comment:
Shouldn't we keep the old behaviour as default and mark it as deprecated for 1 or 2 versions? We can remove it later on. 
Setting it to true will definitely merge all the schemas. But it might impact users without any intimation.

Reply:
Since this is the behavior we want to support by default in the long term, we should enable it and have an “out” while people adjust. If it’s not on, people are not going to turn it on, so we’re not going to be able to see any issues or adjustments we may need to do.

Reply:
Fair point. Turning it on by default.

Reply:
Turned on by default.